test/built-ins/Object/assign/ObjectOverride-sameproperty.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*--
+description: Object properties are assigned to target in ascending index order,
+			 i.e. a later assignment to the same property overrides an earlier assignment.
+es6id:  19.1.2.1
+includes: [assert.js]
```

Comment:
This helper file is included by default, so an explicit entry in `includes` isn't necessary. We just recently removed this pattern from the project (see https://github.com/tc39/test262/pull/309), so it would be good to avoid adding it back.